src/patternfly/demos/DataTable/docs/code.md

```diff
@@ -0,0 +1,3 @@
+<!-- ## Overview -->
+
+This demo's the table component.
```

Comment:
I don't know that you need this but it should be `demos`

Reply:
Going to remove it for consistency